lib/spack/spack/spec.py

```diff
@@ -1901,6 +1899,18 @@ def concrete(self):
         """
         return self._concrete
 
+    def concretize(self, tests: Union[bool, Iterable[str]] = False) -> None:
+        self._dup(self.concretized(tests=tests))
+
+    def concretized(self, tests: Union[bool, Iterable[str]] = False) -> "spack.spec.Spec":
+        import spack.concretize
+
+        msg = "`Spec.concretize` and `Spec.concretized` methods are deprecated and will be "
+        msg += "removed in version 1.0.0. Use `spack.concretize.concretized` instead."
+        tty.warn(msg)
```

Comment:
```suggestion
        warnings.warn(
            "`Spec.concretize` and `Spec.concretized` methods are deprecated and will be "
            "removed in version 1.0.0. Use `spack.concretize.concretized` instead."
        )
```

is less shouty and lets the parser "concatenate"